samples/wire/main.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2018 The Go Cloud Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// The greeter binary simulates an event with greeters greeting guests.
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+func main() {
```

Comment:
If I were coming into this without having the background knowledge of splitting application and setup code, I would be confused as to why `main` is in its own file. Consider moving this into greeter.go.

Reply:
Done.

Reply:
Moved declarations from `greeter.go` into `main.go`.